packages/core/config/webpack.common.js

```diff
@@ -125,6 +129,15 @@ const getStyleLoaders = (isEnvProduction) => {
 const getSharedPlugins = (isEnvProduction) => {
   const shouldIncludeFastRefresh =
     redwoodConfig.web.fastRefresh !== false && !isEnvProduction
+
+  const devTimeAutoImports = isEnvProduction
+    ? {}
+    : {
+        mockGraphQLQuery: ['@redwoodjs/testing/web', 'mockGraphQLQuery'],
+        mockGraphQLMutation: ['@redwoodjs/testing/web', 'mockGraphQLMutation'],
+        mockCurrentUser: ['@redwoodjs/testing/web', 'mockCurrentUser'],
+      }
+
```

Comment:
Instead of adding conditions to exclude from Production, why is FastRefresh in webpack.common instead of being in webpack.development?

Reply:
Because these file names are weird. Common refers to "dev+prod" but also "dev+storybook" 🤷‍♂️

Reply:
Ah, and we don't want this for Storybook. Blast... as you know as well as I these conditionals inside config just start to drive me 🥜

Ok, for improving another time in the future. Thanks!